tests/conftest.py

```diff
@@ -44,7 +44,7 @@
     "h_jwt_client_secret": "TEST_JWT_CLIENT_SECRET",
     "h_authority": "TEST_AUTHORITY",
     "h_api_url_public": "https://example.com/api/",
-    "h_api_url_private": "https://private.com/api/",
+    "h_api_url_private": "https://example.com/private/api/",
```

Comment:
Yeah... uhm I checked what `private.com` is. I would suggest not doing that in public.